CIF-1060 - Implement the log in process (SignIn component)

dplaton opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description

Create the "My Account" component and add it to all the templates as a child node of the navigation panel.
<!--- Describe your changes in detail -->

## Related Issue

CIF-1060

## Motivation and Context

<!--- Why is this change required? What problem does it solve? -->

## How Has This Been Tested?

Functional testing

## Screenshots (if appropriate):

## Types of changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [X] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [X] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [X] I ran all tests locally and they pass.

dplaton committed:
CIF-1060 - Implement the log in process (SignIn component)

Create the "My Account" component and add it to all the templates as a child node of the navigation panel.

mhaack reviewed (commented):
Can we re-think the component naming? Assuming we will have a real my account page and some component(s) (even that does not exist in today's Venia). How about `mini-account` or similar?

dplaton committed:
CIF-1060 - Implement the log in process (SignIn component)

Follow-up on the review:
- rename the component to `miniaccount` and update the references

dplaton committed:
Merge branch 'master' into issue/CIF-1060

dplaton committed:
Merge branch 'issue/CIF-1060' of github.com:adobe/aem-cif-project-archetype into issue/CIF-1060
